src/store/getters/state-getters.ts

```diff
@@ -633,6 +620,12 @@ export const hasBusinessTypeChanged = (state: StateIF): boolean => {
   return (currentEntityType !== originalLegalType)
 }
 
+/** Whether business start date has changed. */
+export const hasBusinessStartDateChanged = (state: StateIF): boolean => {
+  // *** FUTURE: implement this
```

Comment:
@lambert-alex

Reply:
I know you implemented a getter for this, but I named this one consistently with the other business data.

Reply:
Okay once yours is merged I will add this in with my getter.